cmd/cody-gateway/internal/actor/productsubscription/enterpriseportal/enterpriseportal.go

```diff
@@ -26,7 +44,8 @@ func Dial(ctx context.Context, logger log.Logger, addr *url.URL, ts oauth2.Token
 	if insecureTarget {
 		opts = defaults.DialOptions(logger, creds)
 	} else {
-		opts = defaults.ExternalDialOptions(logger, creds)
+		opts = defaults.ExternalDialOptions(logger,
+			append([]grpc.DialOption{creds}, cloudRunDialOptions...)...)
```

Comment:
any reason to not do it this way?
```suggestion
		opts = defaults.ExternalDialOptions(logger, append(cloudRunDialOptions, creds)...)
```

Reply:
It matches the callsite above where `creds` is first 😆

Reply:
Let's keep this as is and test it out - if it helps, I want to move it into the shared package as a "these are options you should use for interacting with MSP services"